csrc/mutator.cpp

```diff
@@ -84,7 +84,8 @@ void OptOutMutator::mutate(Val* s) {}
 
 void OptOutMutator::mutate(NamedScalar* ns) {}
 
-void OptOutMutator::mutate(IterDomain* id) {
+void OptOutMutator::mutate(IterDomain* orig_id) {
```

Comment:
sorry I'm not following the code change here.

Reads like this is an orthogonal issue in mutator that's patched in this PR? My usual ask is, can we have an accompanying test / or a small segment in the PR description to help me understand it? :bow:

Reply:
Good point. I will add that.